### ISSUE TITLE
disable service

### ISSUE THREAD

**Issue by userbradley** [userbradley/documentation.breadnet.co.uk#269]
```
gcurl -d '{ "disableDependentServices": true,}' "https://serviceusage.googleapis.com/v1/projects/<project-number>/services/bigquery.googleapis.com:disable"
{
  "name": "operations/acat.p17-771ae1a8-0233-481b-b515-9c953fd66997
"
}
```

**Comment by userbradley** [comment]
page should include the below

```text
Please specify disable_dependent_services=true if you want to proceed with disabling all services.
```

**Comment by userbradley** [comment]
Closed by #271
